sdks/swagger/sdk.swagger.json

```diff
@@ -229,7 +234,16 @@
           "200": {
             "description": "A successful response.(streaming responses)",
             "schema": {
-              "$ref": "#/x-stream-definitions/sdkGameServer"
```

Comment:
#/x-stream-definitions is not supported by latest `protoc-gen-swagger` and `protoc-gen-openapiv2`. It moved to inline definitions.
https://github.com/grpc-ecosystem/grpc-gateway/pull/1112/files